core/internal/sdata/schema.go

```diff
@@ -31,6 +31,8 @@ type DBSchema struct {
 	edgesIndex        map[string][]edgeInfo   // edges index
 	allEdges          map[int32]TEdge         // all edges
 	relationshipGraph *util.Graph             // relationship graph
+	allowedSchemas    map[string]bool         // track allowed schemas
```

Comment:
This is question, why are we keeping this as a map rather than a list. Aren't with implicitly stating that all the `allowedSchemas` are true ?